zebra/table_manager.c

```diff
@@ -253,7 +253,7 @@ int release_daemon_table_chunks(struct zserv *client)
 	RB_FOREACH (vrf, vrf_name_head, &vrfs_by_name) {
 		zvrf = vrf->info;
 
-		if (!zvrf)
+		if (!zvrf || !zvrf->tbl_mgr)
```

Comment:
This is not the actual problem. Please, take a look at https://github.com/FRRouting/frr/pull/9965.
The same NULL-dereference may happen in `assign_table_chunk` too.